reana_client/cli/workflow.py

```diff
@@ -266,8 +266,8 @@ def workflow_create(ctx, file, name,
         logging.debug(traceback.format_exc())
         logging.debug(str(e))
         click.echo(
-            click.style('Workflow could not be created: \n{}'
-                        .format(str(e)), fg='red'),
+            click.style('Workflow {} could not be created: \n{}'
+                        .format(name, str(e)), fg='red'),
```

Comment:
While re-reading https://github.com/reanahub/reana-client/issues/372:
> We could reformulate the passive tense ("Workflow foo.bar cannot be...") to active tense ("Cannot delete workflow foo.bar.") in case the workflow names are long.

I think this would read better and faster, if we keep it like it is right now the most important part of the phrase will be pushed back to the end when the name is very big, what do you think?